hash/monon_color.xml

```diff
@@ -12,35 +12,35 @@ license:CC0
     Zombie Hunter                                  Jiāngshī lièrén                             Y       Y            NO. 107     ACT             僵尸猎人
     IELTS Adventure                                Yǎsī tǎ dà màoxiǎn                          N       Y            NO. 108     PUZ/EDU         雅思塔大冒险
     Ultimate Brain Power                           Nǎolì liánlián kàn                          N       Y            NO. 109     PUZ/EDU         脑力连连看
-    Locke Kingdom: Big Adventure                   Luòkè wángguó-dà màoxiǎn                    Y       Y            NO. 201     ACT             洛克王国-大冒险
-    Locke Kingdom RPG: Ice Blue                    Luòkè wángguó-bīng zhī lán                  Y       Y            NO. 202     RPG             洛克王国-冰之蓝
-    Locke Kingdom RPG: Black Fire                  Luòkè wángguó-hēi zhī yán                   Y       Y            NO. 203     RPG             洛克王国-黑之炎
-    Locke Kingdom: Card Wars                       Luòkè wángguó-kǎ pái zhàn jì                Y       Y            NO. 204     RPG             洛克王国-卡牌战纪
+    Roco Kingdom: Big Adventure                    Luòkè wángguó-dà màoxiǎn                    Y       Y            NO. 201     ACT             洛克王国-大冒险
+    Roco Kingdom RPG: Ice Blue                     Luòkè wángguó-bīng zhī lán                  Y       Y            NO. 202     RPG             洛克王国-冰之蓝
+    Roco Kingdom RPG: Black Flame                  Luòkè wángguó-hēi zhī yán                   Y       Y            NO. 203     RPG             洛克王国-黑之炎
+    Roco Kingdom: Card Wars                        Luòkè wángguó-kǎ pái zhàn jì                Y       Y            NO. 204     RPG             洛克王国-卡牌战纪
     Armor Warrior                                  Kǎijiǎ yǒngshì dàluàn dòu                   Y       Y            NO. 205     ACT             铠甲勇士大乱斗
-    Locke Kingdom: The King's Badge                Luòkè wángguó-wángzhě huīzhāng              Y       Y            NO. 206     RPG             洛克王国-王者徽章
+    Roco Kingdom: The King's Badge                 Luòkè wángguó-wángzhě huīzhāng              Y       Y            NO. 206     RPG             洛克王国-王者徽章
     Laboratory Mathematica Olympiad                Àoshù shíyàn shì                            N       Y            NO. 207     PUZ/EDU         奥数实验室
     Logic Fight                                    Luójí pīn yī pīn                            Y       Y            NO. 301     PUZ/EDU         逻辑拼一拼
-    Pleasant goat and big big wolf's comet battle  Xǐyángyáng yǔ huītàiláng-juézhàn miē xīng   Y       Y            NO. 302     PUZ/EDU         喜羊羊与灰太狼-决战咩星
+    Pleasant Goat and Big Big Wolf's comet battle  Xǐyángyáng yǔ huītàiláng-juézhàn miē xīng   Y       Y            NO. 302     PUZ/EDU         喜羊羊与灰太狼-决战咩星
     Dragon Ball: The Martial Arts Conference       Lóngzhū-wǔdào dàhuì                         Y       Y            NO. 303     ACT             龙珠-武道大会
```

Comment:
Dragon Ball Budokai games all use the Japanese loanword “Budokai” in English titles rather than translating it (“Martial Arts Meet” is a better literal translation, anyway).